sdk/python/feast/job_service.py

```diff
@@ -145,6 +170,78 @@ def GetJob(self, request, context):
         return GetJobResponse(job=self._job_to_proto(job))
 
 
+def get_expected_job_hash_to_table_refs(client) -> Dict[str, Tuple[str, str]]:
```

Comment:
You are missing a type definition here and a complete docstring. Also, should this be a public method?

Reply:
Probably shouldn't be a public method. Do we need all functions to be compliant to the generated docstrings (with parameters & return type). Sometimes it's overly verbose when the function is simple to describe.

Reply:
We don't have to follow it religiously. For simple methods we can just have a sentence explaining the method.

Reply:
Got it, I'll add a more extensive comment since I'm moving this to the client and modifying it a little.